hack/deploy.sh

```diff
@@ -0,0 +1,274 @@
+#!/bin/bash
```

Comment:
I could remove all the functions and just have chunks of bash. This style help with manual test though, and, IMHO makes the primary functions easier to read.